SwiftDataTables/Classes/Models/DataTableConfiguration.swift

```diff
@@ -9,6 +9,52 @@
 import Foundation
 import UIKit
 
+public enum DataStyles {
+    
+    public enum Colors {
+        
+        public static var highlightedFirstColor: UIColor = {
+            return setupColor(normalColor: UIColor(red: 0.941, green: 0.941, blue: 0.941, alpha: 1),
+                              darkColor: UIColor(red: 0.16, green: 0.16, blue: 0.16, alpha: 1),
+                              defaultColor: UIColor(red: 0.941, green: 0.941, blue: 0.941, alpha: 1))
+        }()
+        
+        public static var highlightedSecondColor: UIColor = {
+            return setupColor(normalColor: UIColor(red: 0.9725, green: 0.9725, blue: 0.9725, alpha: 1),
+                              darkColor: UIColor(red: 0.13, green: 0.13, blue: 0.13, alpha: 1),
+                              defaultColor: UIColor(red: 0.9725, green: 0.9725, blue: 0.9725, alpha: 1))
+        }()
+        
+        public static var unhighlightedFirstColor: UIColor = {
+            return setupColor(normalColor: UIColor(red: 0.9725, green: 0.9725, blue: 0.9725, alpha: 1),
+                              darkColor: UIColor(red: 0.06, green: 0.06, blue: 0.06, alpha: 1),
+                              defaultColor: UIColor(red: 0.9725, green: 0.9725, blue: 0.9725, alpha: 1))
+        }()
+        
+        public static var unhighlightedSecondColor: UIColor = {
+            return setupColor(normalColor: .white,
+                              darkColor: UIColor(red: 0.03, green: 0.03, blue: 0.03, alpha: 1),
+                              defaultColor: .white)
+        }()
+    }
+}
+
+public let Style = DataStyles.self
```

Comment:
What's the use of this property? This property isn't used anywhere.

Reply:
“Style” property use for call DataStyles, but I really don’t use it in code. It can be safe removed or refactored to use this.
Example: Style.Colors.etc instead DataStyles.Colors.etc, but I think, DataStyles looks pretty good.